modules/creating-oci-resources-services.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-oci-resources-services_{context}"]
+= Creating OCI resources and services
+
+Create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization requirements.
```

Comment:
organization's? 

```suggestion
Create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization's requirements.
```